spring-cloud-skipper-server-core/src/test/java/org/springframework/cloud/skipper/server/service/ArgumentSanitizerTests.java

```diff
@@ -48,7 +48,7 @@ public void testPasswordApps() throws Exception {
 				TestResourceUtils.qualifiedResource(getClass(), "passwordredacted.yaml").getInputStream(),
 				Charset.defaultCharset());
 		String result = ArgumentSanitizer.sanitizeYml(initialYaml);
-		assertThat(result).isEqualTo(redactedYaml);
+		assertThat(result.replace("\r\n", "\n")).isEqualTo(redactedYaml.replace("\r\n", "\n"));
```

Comment:
Can we please create a utility that does this lenient comparison? Its proliferating around now.

Reply:
K. Do you mind creating a test util that will do this comparison, maybe something like:

```java
void assertEqualsReplacingCrlf(String, String)
```
so that we have this logic in one place?

Reply:
The changes was in two modules. So I added a function in one test and a utility class in module with many usages.